site/docs/components/slider/accessibility.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Cursor interaction
+
+Allow cursor interaction of a handle by increasing the target area around the indicator to -size-base. This creates a more accessible click/touch area which meets WCAGs minimum Target Size (2.5.5) standard.
+
+### Tooltip
+
+Tooltip displays the value on handle focus. Tooltip remains persistent on cursor drag or moving the handle along the track with keyboard interaction. By default, tooltip is configured to flip to switch placement sides if constrained by space, so that it remains visible.
```

Comment:
This doesn't read as a11y guidance?

Reply:
Have now incorporated in the first example description.